src/models/transfer.js

```diff
@@ -116,7 +116,7 @@ PersistentModelMixin(Transfer, sequelize, {
   ledger: Sequelize.STRING(1024),
   debits: JsonField(sequelize, 'Transfer', 'debits'),
   credits: JsonField(sequelize, 'Transfer', 'credits'),
-  part_of_payment: Sequelize.STRING(1024),
+  additional_info: JsonField(sequelize, 'Transfer', 'additional_info'),
```

Comment:
> You could argue that file metadata is part of the file's data because they are stored together on the disk

If we're talking about actual filesystem metadata, that is not at all stored together with the data. In NTFS the metadata is stored in the MFT and in ext* it's stored in the [inode table](https://ext4.wiki.kernel.org/index.php/Ext4_Disk_Layout#Inode_Table).

That's my point, if this data we're talking about was something that the ledger generated *about* the Transfer object, then I would call it metadata. As is, it's custom fields that you're adding into the object. 

All naming is ultimately a matter of taste. I would say unless somebody has a strong objection, I'm happy for us to merge the PR as-is.

LGTM.